bin.src/imsim.py

```diff
@@ -72,9 +76,10 @@
 if args.psf_file is None or not os.path.isfile(args.psf_file):
     psf = desc.imsim.make_psf(args.psf, obs_md, log_level=args.log_level)
     if args.psf_file is not None:
-        desc.imsim.save_psf(psf, args.psf_file)
+        desc.imsim.save_psf(psf, args.psf_file, args.psf_screen_file)
 else:
-    psf = desc.imsim.load_psf(args.psf_file, log_level=args.log_level)
+    psf = desc.imsim.load_psf(args.psf_file, args.psf_screen_file,
```

Comment:
I'd recommend explicitly naming the kwarg here: `screen_file=args.psf_screen_file`.
IMO, it is good practice to use the kwarg name when an argument is optional, and limit positional arguments to required arguments with no default.  This tends to lead to less breakage if someday additional optional arguments are added in the future, which might mess up the ordering.